naresh/src/main/java/com/lftechnology/phpjava/ems/utlis/DbFactory.java

```diff
@@ -0,0 +1,76 @@
+package com.lftechnology.phpjava.ems.utlis;
+
+import com.lftechnology.phpjava.ems.constants.Constant;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+/**
+ * DbFactory
+ *
+ * @author Naresh Maharjan <nareshmaharjan@lftechnology.com>
+ * @since August, 08 2016
+ */
+public abstract class DbFactory {
+    private static Properties properties = new Properties();
+    private static Connection connection = null;
+
+
+    /**
+     * load database configuration for properties file
+     *
+     * @return
+     * @throws IOException
+     * @author Naresh Maharjan <nareshmaharjan@lftechnology.com>
+     */
+    private static void loadProperties() {
+
+        InputStream input = null;
+
+        try {
+            String filePath = "src/main/java/com/lftechnology/phpjava/ems/configs/db.properties";
```

Comment:
Does this code works in case of windows system? how would you make this platform independent?